code/modules/clothing/head/misc_special.dm

```diff
@@ -232,19 +233,17 @@
 /obj/item/clothing/head/wig/update_icon_state()
 	var/datum/sprite_accessory/S = GLOB.hairstyles_list[hairstyle]
 	if(!S)
-		icon = 'icons/obj/clothing/hats.dmi'
 		icon_state = "pwig"

```

Comment:
this should just be wig now, no?

Reply:
actually I guess there's no need for this since the icon state does not change anymore, I can just return instead